### ISSUE TITLE
OpenMC required for tests

### ISSUE THREAD

**Issue by yadudoc** [watts-dev/watts#61]
I'm over from JOSS looking at this repo to make sure everything checks out. I've looked over the docs quickly and everything looks really clean and understandable. I need to get the tests running on my laptop and I'm running into some issues that I'm hoping to get help with.

I've installed  `watts` from source, and when I run `pytest tests` I get the following (relevant snippet):
```
ImportError while importing test module '/tmp/watts/tests/test_plugin_openmc.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_plugin_openmc.py:7: in <module>
    import openmc
E   ModuleNotFoundError: No module named 'openmc'
```

I'm using conda, so I followed the install instructions here `https://docs.openmc.org/en/stable/quickinstall.html` and installed from mamba, and even still I get the same error. I've checked with `python3 -c "import openmc; print(openmc.__version__)"` to confirm that openmc is indeed installed. Any advice on this is much appreciated.

Linking [https://github.com/openjournals/joss-reviews/issues/4735]

**Comment by paulromano** [comment]
@yadudoc The only thing I can think is that when you call `pytest`, it's picking it up from a different Python environment, not the conda environment that has OpenMC installed in it. Perhaps try running `python3 -m pytest tests` instead?

**Comment by yadudoc** [comment]
@paulromano That definitely helped. I'm getting new errors now and the error message and troubleshooting guide over on OpenMC seems to indicate that I'm missing the `OPENMC_CROSS_SECTIONS` env var and an XML file.

Have you considered dropping a mock XML file into tests and setting the required env var from conftests? 
Here's a snippet of the error:

```
tests/test_plugin_openmc.py FFF  
...
                error_msg = 'OpenMC aborted unexpectedly.'
            error_msg = ' '.join(error_msg.split())
    
>           raise RuntimeError(error_msg)
E           RuntimeError: No cross_sections.xml file was specified in materials.xml or in the OPENMC_CROSS_SECTIONS environment variable. OpenMC needs such a file to identify where to find data libraries. Please consult the user's guide at https://docs.openmc.org/ for information on how to set up data libraries.

/home/yadu/anaconda3/envs/watts/lib/python3.10/site-packages/openmc/executor.py:116: RuntimeError

```

**Comment by paulromano** [comment]
Closed with the merge of #66
